doc/index.rst

```diff
@@ -1,8 +1,23 @@
 IPv8 Documentation
 ==================
 
-Welcome to the IPv8 documentation!
-This file will help you navigate the existing documentation.
+Welcome to the IPv8 documentation! This file will help you navigate the existing documentation.
+The documentation will explain the concepts behind IPv8, the architecture, and guide you to making your first overlay.
+
+IPv8 Features
```

Comment:
I spotted some partial truths and some incorrect claims regarding the IPv8 functionality. Some of the claims are only true for Tribler and others are only true for the TunnelCommunity:

- `encrypted network connection`  〰️ 
- `to a known person` 〰️ 
- `robust against various network problems and security issues` 〰️
- `is a networking layer` ✔️ 
- `which offers identities` ✔️ 
- `exposes reputation functions` ❌ 
- `storage of reputation data` ❌ 
- `calculates reputation levels`  ❌ 
- `IPv8 is a mechanism designed to build trust`  ❌
- `offers public-key based identities` ✔️ 
- `offers protection against eavesdropping`   〰️
- `man-in-the-middle attacks` 〰️
- `peer failure` ❌
- `network failure` 〰️
- `packet loss` ❌
- `change of IP numbers, network mobility` ✔️ 
- `blocking by NAT/Firewalls` ✔️

Reply:
Very handy ToDo list, thnx! 
Addressed both all errors and also all half truths.